### PR TITLE
Adding recipe for bio-hansel

### PR DESCRIPTION
**Pull request by hellothisisMatt**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jenzopr** [comment]
Thanks @hellothisisMatt for your contribution!
Currently, I see a slight contradiction between the `build.sh` and the source given in `meta.yaml`. With the setup you propose, it is not for sure that the announced version in `meta.yaml` actually comes via `pip install` (e.g. if the upstream PyPI version changes). 
It would be a lot easier if you entirely rely on the PyPI package and omit the git source. You can create a skeleton `meta.yaml` and `build.sh` by executing `conda skeleton pypi bio_hansel`. Have a look at https://bioconda.github.io/guidelines.html#python for help.
Best,
Jens

**Comment by hellothisisMatt** [comment]
Hello @jenzopr, thank you very much your help!
I now used `conda skeleton pypi bio_hansel` and fixed my build.sh and meta.yaml files to only rely on PyPI.

File: recipes/bio_hansel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  fn: bio_hansel-0.1.0.tar.gz
+  url: https://pypi.python.org/packages/36/21/07f9c1e48794ea712d5496fb58cfea3ca8539f16ea04cf6e2e78c9309560/bio_hansel-0.1.0.tar.gz
+  md5: c7178ecd9e0f30e7669fe497188d7e49
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python >=3.5
```
Comment (bgruening) [inline recipes/bio_hansel/meta.yaml]: can you remove the pinning here and inlucde under the build section `skip: True  # [py2k]`

File: recipes/bio_hansel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - pip
```
Comment (bgruening) [inline recipes/bio_hansel/meta.yaml]: pip is probably not needed

File: recipes/bio_hansel/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - setuptools
+    - numpy >=1.12.1
+    - pandas >=0.20.1
+    - attrs
+
+  run:
+    - python >=3.5
```
Comment (bgruening) [inline recipes/bio_hansel/meta.yaml]: please remove the pinning here

File: recipes/bio_hansel/meta.yaml (hunk 1 above)
Comment (hellothisisMatt) [reply recipes/bio_hansel/meta.yaml]: @bgruening Thanks for reviewing my changes! When you mention `pinning` do you mean the `>=3.5` part?

Comment (bgruening) [reply recipes/bio_hansel/meta.yaml]: Oh, yes :)

Comment (hellothisisMatt) [reply recipes/bio_hansel/meta.yaml]: Awesome, I'll remove `>=3.5`. Thanks @bgruening.

**Comment by jenzopr** [comment]
Great work @hellothisisMatt ! Thanks for your additional comments, @bgruening !
Btw. and in general: What do you think about the non-canonical source URL for PyPI? E.g. conda-forge wants it like `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`..

**Comment by bgruening** [comment]
@hellothisisMatt I like the proposal from @jenzopr. Do you want to use this?

**Comment by hellothisisMatt** [comment]
@bgruening I like the idea too! I'll make the appropriate changes.

**Comment by hellothisisMatt** [comment]
Okay, the URL is now standard, and I also changed the hashing to match as well.

**Comment by jenzopr** [comment]
Well done! Thanks a lot

**Comment by bgruening** [comment]
Perfect! Thanks a lot!

**Comment by hellothisisMatt** [comment]
Thank you both for the help!

**Pull request merged**